src/SFGUI/RenderQueue.cpp

```diff
@@ -8,6 +8,8 @@
 #include <SFML/Graphics/Sprite.hpp>
 #include <SFML/Graphics/Text.hpp>
 
```

Comment:
There shouldn't be a space between SFML and C++ header blocks :stuck_out_tongue:.